lib/parsers/bmp_parser.rb

```diff
@@ -0,0 +1,41 @@
+class FormatParser::BMPParser
+  include FormatParser::IOUtils
+
+  VALID_BMP = 'BM'
+  PIXEL_ARRAY_OFFSET = 54
+  COLOR_TYPES = {
+    1 => :monochrome
+  }
+
+  # Based on https://en.wikipedia.org/wiki/BMP_file_format
```

Comment:
The comment is kinda hanging in between - it doesn't belong to the class, neither does it belong to the method underneath due to the spacing. Maybe we could put it at the top of the file?